PyDSS/utils/dss_utils.py

```diff
@@ -22,13 +27,54 @@ def read_pv_systems_from_dss_file(filename):
     """
     New PVSystem.pv_1114018 bus1=133294_xfmr.1.2 phases=2
     """
-    regex = re.compile(r"New (PVSystem\.\w+)\s", re.I)
+    regex = re.compile(r"New (PVSystem\.)(\w+)\s", re.I)
 
     with open(filename) as fp_in:
         for line in fp_in:
             match = regex.search(line)
             if match:
-                pv_systems.append(match.group(1))
+                pv_systems.append(match.group(1) + match.group(2).lower())
 
     logger.debug("Found pv_systems=%s in %s", pv_systems, filename)
     return pv_systems
```

Comment:
This is not specific to SMART-DS. It will work with any PVSystem name as long as that name is alphanumeric. Can the names have special characters ("-", ":", etc)?

This code gets used in a CLI command. Here is the documentation for the use case:

Assign element names to controllers. This example will add all PVSystems defined in an OpenDSS input file to the default Volt-Var PvController.
```
pydss edit-scenario -p ./project -s scenario1 update-controllers -t PvController -f ./project/DSSfiles/PVGenerators_existing_VV.dss -c volt-var
```